dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/Governors/Standard/Generic/GovCT2.mo

```diff
@@ -0,0 +1,352 @@
+  connect(lastValue.y, multiSwitchNoVector.u3) annotation(
+    Line(points = {{114, -152}, {104, -152}, {104, -180}, {-168, -180}, {-168, -162}}, color = {0, 0, 127}));
+  connect(limitValveMaxValveMin.y, lastValue.u) annotation(
+    Line(points = {{142, -88}, {150, -88}, {150, -152}, {136, -152}}, color = {0, 0, 127}));
+  connect(minDm.y, expOmegaToTheDm.u2) annotation(
+    Line(points = {{156, 158}, {174, 158}, {174, 144}, {202, 144}}, color = {0, 0, 127}));
+  connect(omegaRefPu, add.u1) annotation(
+    Line(points = {{-332, 24}, {-280, 24}, {-280, -12}, {-256, -12}}, color = {0, 0, 127}));
+  annotation(
+    preferredView = "diagram",
+    uses(Modelica(version = "3.2.3")),
+    Documentation(info = "<html><head></head><body>This generic governor model (CIM name GovCT2) can be used to represent a variety of prime movers controlled by PID governors. For more information, see IEC 61970-302.</body></html>"),
+    Diagram(coordinateSystem(extent = {{-320, -200}, {320, 220}}), graphics = {Text(origin = {248, -35}, extent = {{-7, -3}, {7, 3}}, textString = "cfe"), Text(origin = {158, -84}, extent = {{-17, -4}, {17, 4}}, textString = "fsr"), Rectangle(origin = {157, 14}, lineColor = {0, 0, 255}, lineThickness = 0.75, extent = {{-41, 68}, {41, -68}}), Text(origin = {174, 45}, lineColor = {0, 0, 255}, extent = {{-23, -21}, {23, 21}}, textString = "frequency-
+dependent
+limit"), Text(origin = {135, -194}, extent = {{-44, -4}, {44, 4}}, textString = "valve stroke"), Text(origin = {85, -184}, extent = {{-30, -4}, {30, 4}}, textString = "governor output"), Rectangle(origin = {70, -20}, lineColor = {0, 0, 255}, lineThickness = 0.75, extent = {{-24, 86}, {24, -86}}), Text(origin = {-22, 120}, extent = {{-13, -4}, {13, 4}}, textString = "Texm"), Text(origin = {106, 125}, extent = {{-11, -3}, {11, 3}}, textString = "Tex"), Text(origin = {-70, 156}, extent = {{-13, -4}, {13, 4}}, textString = "tlim"), Text(origin = {73, 44}, lineColor = {0, 0, 255}, extent = {{-20, -18}, {20, 18}}, textString = "Low
+Value
+Select"), Text(origin = {182, 192}, extent = {{-17, -4}, {17, 4}}, textString = "dm>=0"), Text(origin = {168, 164}, extent = {{-17, -4}, {17, 4}}, textString = "dm<=0"), Text(origin = {-190, -21}, extent = {{-19, -1}, {19, 1}}, textString = "deltaOmega"), Text(origin = {-106, -15}, extent = {{-19, -1}, {19, 1}}, textString = "acceleration")}),
+    Icon(coordinateSystem(extent = {{-320, -200}, {320, 220}}), graphics = {Text(origin = {7, 7}, extent = {{-279, 123}, {279, -123}}, textString = "GovCT2"), Rectangle(origin = {0, 10}, extent = {{-320, 210}, {320, -210}})}),
+    experiment(StartTime = 0, StopTime = 10.5, Tolerance = 0.001, Interval = 0.002),
+    __OpenModelica_simulationFlags(lv = "LOG_STATS", s = "rungekutta"),
+    __OpenModelica_commandLineOptions = "--matchingAlgorithm=PFPlusExt --indexReductionMethod=dynamicStateSelection -d=initialization,NLSanalyticJacobian");
+end GovCT2;
```

Comment:
Could you organise the model as in drawing a rectangle to specify the three parts : 
- main control path  (resulting in fsrn)
- Acceleration command (resulting in fsra)
- Temperature command (resulting in fsrt)

You could also increase the white Diagram size to have more space